src/api/json/catalog.json

```
@@ -4232,11 +4232,47 @@
     },
     {
       "name": "StrmPrivacy batch job configuration file",
-      "description": "StrmPrivacy derived from the BatchJob protobuf definition",
+      "description": "StrmPrivacy batch job (https://docs.strmprivacy.io/docs/latest/concepts/data-processing/batch-jobs/)",
```

```suggestion
      "description": "StrmPrivacy batch job\nhttps://docs.strmprivacy.io/docs/latest/concepts/data-processing/batch-jobs/",
```

Optional fix: doc url can be placed on a separate line.